src/linux/link.rs

```diff
@@ -168,11 +172,141 @@ fn call_patchelf(elf_path: &Path, new_rpath: &[PathBuf]) -> Result<(), RelinkErr
     }
 }
 
+fn ctx(object: &Elf) -> goblin::container::Ctx {
```

Comment:
It is not clear why this is needed at first glance, let's document this function!